packages/writing-style/styles/commercetools/WordList-Suggestions.yml

```diff
@@ -0,0 +1,8 @@
+extends: substitution
+message: "Consider using '%s' instead of '%s' (case sensitive)."
+level: suggestion
+ignorecase: false
```

Comment:
I think these very loose rules that are just ideas should be case insensitive.  e.g. the extension case would typically be lowercase anyways